include/pybind11/buffer_info.h

```diff
@@ -44,10 +44,12 @@ struct buffer_info {
     void *ptr = nullptr;          // Pointer to the underlying storage
     ssize_t itemsize = 0;         // Size of individual items in bytes
     ssize_t size = 0;             // Total number of entries
-    std::string format;           // For homogeneous buffers, this should be set to format_descriptor<T>::format()
+    std::string format; // For homogeneous buffers, this should be set to
```

Comment:
FWIW, for these sorts of things, I'd do something like:

```
<empty line>
// Comment
std::string thing;
```

That will format much better, remain readable, and use less horizontal space for some extra vertical space.

No need to do that now. But that would be my general solution.

Reply:
Totally agree.

Reply:
I was going back and forth myself exactly here, then decided clang-format is actually doing a very nice job, preserving the original intent (no semi-pun intended). It's not obvious here, but if you look under #3713 it's great there. — But I'm happy to change it around. What do you prefer? Put only some comments on top, as needed? Or all, systematically?

Reply:
Let's go with that then. Merge at your discretion.